docs/content/SRS/SRS.md

```diff
@@ -261,23 +379,45 @@ _The following Software Requirements Specification for `tinymorph` is using [Vol
 
 ### 15.1 Access Requirements
 
-[Insert your content here.]
+SR-AC1. The application will use secure environment variables to manage configuration settings that include sensitive information.
+
+Rationale: Managing settings through environment variables keeps sensitive information out of the codebase, reducing the risk of exposure and enhancing security.
 
 ### 15.2 Integrity Requirements
 
-[Insert your content here.]
+SR-INT1. All communication between the client UI, backend services, and external APIs must be encrypted using HTTPS.
+
+Rationale: HTTPS encryption secures data in transit, preventing interception or tampering. It also ensures the confidentiality and integrity of user data and commands.
+
+SR-INT2. Implement DNS security measures to ensure that DNS queries and responses are protected against tampering and spoofing.
+
+Rationale: Securing DNS interactions prevents attackers from manipulating or rerouting network traffic. This is critical for maintaining the integrity of application data.
+
+SR-INT3. The application will use content security policies to mitigate the risk of XSS attacks.
+
+Rationale: Content Security Policies (CSP) are an effective security control to prevent XSS attacks by restricting the sources from which scripts can be loaded and executed in the application. CSP will help in safeguarding against data theft and maintain the integrity of the content delivered to users.
 
 ### 15.3 Privacy Requirements
 
-[Insert your content here.]
+SR-P1. The application must ensure that it does not collect or store personal information, adhering strictly to privacy by design principles.
+
+Rationale: By not collecting personal information, the application minimizes privacy risks and complies with privacy laws and regulations. Avoiding personal data storage also reduces the need for complex data security measures, allowing the project to focus more on enhancing user experience and functionality.
 
 ### 15.4 Audit Requirements
 
-[Insert your content here.]
+SR-AU1. Implement monitoring and logging of access to the inference server and application endpoints.
```

Comment:
audit often refers to tolerance and security measures, which often times refers to how the providers are being used, that's all.

Reply:
Put more attention on the requirement for monitoring service providers for compliance with security polices and performance expectations.